lib/renderer/react.js

```diff
@@ -13,12 +13,15 @@ const reactExport = module.exports;
  * @param {string} templateLocation located for thetemplate
  * @param {string} templateContentDir where the template content are located
  * @param {string} transpiledTemplateLocation folder for the transpiled code
+ * @param {Boolean} compile Whether to compile the template files or used the cached transpiled version provided by the template in the '__transpiled' folder
  */
-reactExport.configureReact = async (templateLocation, templateContentDir, transpiledTemplateLocation) => {
+reactExport.configureReact = async (templateLocation, templateContentDir, transpiledTemplateLocation, compile) => {
```

Comment:
You probably want to revoke the extra argument `compile` added here. Currently the 

`        await configureReact(this.templateDir, this.templateContentDir, TRANSPILED_TEMPLATE_LOCATION);
`

in generator.js has syntax error, which expects 4 arguments but your updated codes with a conditional statement only give it three arguments. 

```suggestion
reactExport.configureReact = async (templateLocation, templateContentDir, transpiledTemplateLocation) => {
```